src/components/AboutVisualizationUnit/styles/AboutVisualizationUnit.module.css

```diff
@@ -0,0 +1,56 @@
+.container {
+    padding: var(--spacers-dp16);
+    border-bottom: 1px solid var(--colors-grey400);
+    background-color: var(--colors-white);
+}
+
+.expanded {
+    padding-bottom: var(--spacers-dp32);
+}
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    font-size: var(--spacers-dp16);
+    font-weight: 500;
+    line-height: 21px;
+    color: var(--colors-grey900);
+}
+
+.content {
+    font-size: 14px;
+    line-height: 18px;
+    color: var(--colors-grey900);
+}
+
+.content p {
```

Comment:
generally not a fan of element selectors when we have access to the markup to use class selectors. add a class and target that instead?

Reply:
Fixed.